kerchunk/combine.py

```diff
@@ -443,8 +446,8 @@ def store_coords(self):
         logger.debug("Written coordinates")
         for fn in [".zgroup", ".zattrs"]:
             # top-level group attributes from first input
-            if fn in m:
-                self.out[fn] = ujson.dumps(ujson.loads(m[fn]))
+            if m.fs.exists(fn):
```

Comment:
Ask the zarr store or dict if this exists rather than hitting the fielsystem directly so it is compatible with zarr python 2